assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidget/UserDimensionsPieChart.js

```diff
@@ -142,6 +150,28 @@ export default function UserDimensionsPieChart( { dimensionName, dimensionValue,
 		}
 	}, [ chartID, setValues ] );
 
+	useEffect( () => {
+		const onTooltipClick = ( event ) => {
+			const { target } = event || {};
+			if ( ! target?.classList?.contains( 'googlesitekit-cta-link__tooltip' ) ) {
+				return;
+			}
+
+			const label = target.dataset.rowLabel;
+			if ( label === '(other)' || label === '(not set)' ) {
+				trackEvent( 'all_traffic_widget', 'help_click', label );
+			} else if ( label === 'others' ) {
+				trackEvent( 'all_traffic_widget', 'others_source_click', null );
+			}
+		};
+
+		global.addEventListener( 'click', onTooltipClick );
+
+		return () => {
+			global.removeEventListener( 'click', onTooltipClick );
+		};
```

Comment:
Would it be possible to add/remove this listener maybe on the All Traffic widget pie chart wrapper, rather than the entire DOM? This would be a bit more targeted and slightly improve performance as this function wouldn't need to run for _any_ click on the entire page.

Reply:
Yes, it's a good point. Updated.